content/2.0.html.haml

```diff
@@ -90,109 +89,121 @@ title: "Jenkins 2.0"
+        %li Pipeline as code provides a common language to help teams (e.g. Dev and Ops) work together.
+        %li Easily share pipelines between teams by storing common functions in global libraries
+        
+  %tr
+    %td New Pipeline Stage View
+    %td 
+      %ul
+        %li A new interface addition to make it easy to visualize the progress across an entire pipeline 
+    
+  %tr
+    %td Durable
+    %td
+      %ul
+        %li A new interface addition to make it easy to visualize the progress across an entire pipeline 
+        
+  %tr      
+    %td Native support for Git, GitHub branches
+    %td 
+      %ul
+        %li Each feature branch can have its own delivery pipeline. 
+  
+%h2 Improved Usability
+
+%h3 Problem
+
+%p When setting up a new Jenkins installation users have to filter through and select from the 1100 (and rising) available plugins. Once the plugins are installed users have to navigate and complete the multiple plugin configurations choices. This makes the "Getting Started Experience" time consuming, and for new users can mean going through trial and error before they have a stable installation.
+
+  
+%h3 Solution
+
+%p For new users, Jenkins 2.0 starts off with set of recommended plugins to help get you started with the right set of tools to get up and running with Jenkins quickly.
+
+%p There are a number of useful improvements in user interface, such as an improved configuration page, to make users lives easier.
+
+
+%h4 Key Features and Benefits for Pipelines
+
+%table{:border => 1} 
+  %tr
+    %th Features
+    %th Benefits
+  %tr 
+    %td New getting started experience
+    %td
+      %ul
+        %li Start with a set of recommended plugins that match a users preferred use case. 
+        %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
+          %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
           :width => 550}/
 
-    %p
-      Learn more about this on our
-      %a{:href => '/solutions/pipeline'}
-        Pipeline solutions page
-      !
-
-  %dt
-    Out of the box: Easy to get it right
-  %dd
-    %p
-      For new users, Jenkins 2.0 starts off with set of recommended plugins to
-      help get you started with the right set of tools to get up and running with
-      Jenkins quickly.
-
-      For the more adventurous users, the Jenkins 2.0 initial setup process
-      also allows you to pick and choose exactly the plugins you want to meet
-      your specific needs. The list of plugins is hand-picked by Jenkins experts
-      from our popular plugins to ensure the best getting-started experience for
-      new users.
-
-    %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
-      %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
-          :width => 550}/
+  %tr
+    %td New Configurations tab view
```

Comment:
That is possibly being reworked (scroll spy rather than tabs). Don't go into so much detail here, as it may turn out to be wrong.